man/systemd.netdev.xml

```diff
@@ -911,6 +916,23 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Sequence=</varname></term>
+        <listitem>
+          <para>The <varname>Sequence</varname> is a integer serializes the packets. Only applies for ERSPAN tunnel.
+          Defaults to <literal>unset</literal>.
```

Comment:
Still not clear for me. How about the following (I am not sure the statement is correct or not)?
`<para>Takes a boolean value. If set to yes, then packets are serialized. Only applies for ... </para>`

Reply:
yes sure .